### PR TITLE
Add migration createFields

### PR DESCRIPTION
**Pull request by ochupa**
dev
## JIRA

* [Main JIRA ticket](https://jira.softserve.academy/secure/RapidBoard.jspa?rapidView=id)

## Code reviewers

- [ ] @github_username

### Second Level Review

- [ ] @github_username

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# This file is auto-generated from the current state of the database. Instead
```
Comment (maksymst) [inline db/schema.rb:1]: There is no `fields` table in this file. Did you run `rake db:migrate`?

**Review by maksymst — CHANGES_REQUESTED** [review]

File: db/schema.rb (hunk 1 above)
Comment (ochupa) [reply db/schema.rb:1]: Yes, ran it, but after it saw some pitfalls in migration file and made rollback. After correcting migration file I didn't run it again. As I understand, together with migration file I have to upload updated schema.rb ?

Comment (maksymst) [reply db/schema.rb:1]: Correct

File: db/migrate/20210408145903_create_fields.rb [hunk 2]
```diff
@@ -0,0 +1,20 @@
+class CreateFields < ActiveRecord::Migration[6.1]
+  def change
+    create_table :fields do |t|
+      t.uuid :uuid, null: false
+      t.integer :calculator_id, null: false
```
[the remark this answers is not in the document]
Comment (ochupa) [reply db/migrate/20210408145903_create_fields.rb]: Thanks. I refactored migration file according this guide.

**Comment by maksymst** [comment]
@ochupa I've added one more field into the ticket. Please add it to your migration as well

File: db/migrate/20210408145903_create_fields.rb [hunk 3]
```diff
@@ -0,0 +1,19 @@
+class CreateFields < ActiveRecord::Migration[6.1]
+  def change
+    create_table :fields do |t|
+      t.uuid :uuid, null: false
+      t.references :calculator
```
Comment (maksymst) [inline db/migrate/20210408145903_create_fields.rb:5]: This field as well as `selector`, `type`, and `kind` should have `NOT NULL` constraint

Comment (ochupa) [reply db/migrate/20210408145903_create_fields.rb]: Ok. I made this changes. I'm not sure if I'm doing in correct way every time using "rebase -i" command.

**Review by maksymst — APPROVED** [review]
LGTM

**Pull request merged**